samples/kotlin-android-app/src/main/java/com/appcues/samples/kotlin/ExampleApplication.kt

```diff
@@ -58,5 +60,8 @@ class ExampleApplication : Application() {
                 }
             }
         }
+
+        Appcues.registerCustomComponent("simpleButtonView", SimpleButtonView(this))
```

Comment:
should we remove this one and just have the `liveStream` to match iOS?

Reply:
we can remove for now, eventually we would want to add more examples